content/docs/alerting/configuration.md

```diff
@@ -507,3 +512,29 @@ endpoint:
 There is a list of
 [integrations](/docs/operating/integrations/#alertmanager-webhook-receiver) with
 this feature.
+
+## `<wechat_config>`
+
+Wechat notifications are sent via the [Wechat
+API](http://admin.wechat.com/wiki/index.php?title=Customer_Service_Messages).
+
+```yaml
+# Whether or not to notify about resolved alerts.
+[ send_resolved: <boolean> | default = true ]
+
+# The API key to use when talking to the Wechat API.
+[ api_secret: <secret> | default = global.wechat_secret_url ]
+
+# The Wechat API URL.
+[ api_url: <string> | default = global.wechat_api_url ]
+
+# The corp id for authentication
+[ corp_id: <string> | default = global.wechat_api_corp_id ]
+
+# API request data as defined by the Wechat API.
+[ message: <string> | default = '{{ template "wechat.default.message" . }}' ]
+[ agent_id: <string> | default = '{{ template "wechat.default.agent_id" . }}' ]
```

Comment:
This doesn't seem right, this field isn't templated so why does it have that default?

Reply:
Yep, you're right.

Ping @stuartnelson3 @berlinsaint @songjiayang

Reply:
Hi, in work WeChat,  a `agent_id ` means a chat room,  so I think `agent_id` should be a categoric value,  no need template, right ?

Reply:
Then I've got https://github.com/prometheus/alertmanager/pull/1253 to remove those.